src/services/S3MHandlersAnalysis/implementations/ProjectProcessor.groovy

```diff
@@ -0,0 +1,72 @@
+package services.S3MHandlersAnalysis.implementations
```

Comment:
Essa classe parece fazer os mesmo da ForkAndEnableTravisProcessor, se realmente for, usar a ForkAndEnableTravisProcessor invés dela no injector